app/views/vm_common/_reconfigure.html.haml

```diff
@@ -97,19 +94,121 @@
+                            "ng-change"                   => "hdModeChanged()",
+                            "data-width"                  => "auto",
+                            "required"                    => "",
+                            "selectpicker-for-select-tag" => "",
+                            "width"                       => "10")
+            %td
+              %input.form-control{"type"        => "text",
+                                  "id"          => "dvcSize",
+                                  "name"        => "dvcSize",
+                                  "ng-model"    => "reconfigureModel.hdSize",
+                                  "ng-pattern"  => "hdpattern",
+                                  "placeholder" => "Enter New Size",
+                                  "ng-change"   => "hdChanged()"}
+              %span{"style" => "color:red", "ng-show" => "rowForm.dvcSize.$invalid"}
+                = _(" Valid numeric disk size required ")
+            %td.narrow
+              = select_tag('hdUnit',
+                       options_for_select(%w(MB GB)),
+                       "ng-model"                    => "reconfigureModel.hdUnit",
+                       "ng-change"                   => "hdUnitChanged()",
+                       "data-width"                  => "auto",
+                       "required"                    => "",
+                       "selectpicker-for-select-tag" => "")
+            %td.narrow
+              %input{"type"     => "checkbox",
+                     "name"     => "cb_dependent",
+                     "ng-model" => "reconfigureModel.cb_dependent"}
+            %td.narrow
+            %td
+              %button.btn.btn-primary.btn-sm{:type => "button", "ng-click" => "addDisk()"}= _(' Save ')
+          %tr{"ng-repeat" => "disk in reconfigureModel.vmdisks", "ng-class" => "{'danger': disk.add_remove == 'remove', 'active': disk.add_remove == 'add'}"}
+            %td
+              {{disk.hdFilename}}
+            %td
+              {{disk.hdType}}
+            %td
+              {{disk.hdMode}}
+            %td
+              {{disk.hdSize}}
+            %td.narrow
+              {{disk.hdUnit}}
+            %td.narrow
+              %input{"type"        => "checkbox",
+                     "name"        => "cb_dependent",
+                     "ng-model"    => "disk.cb_dependent",
+                     "ng-if"       => "disk.add_remove=='add'",
+                     "ng-disabled" => "disk.add_remove=='add'",
+                     "checkchange" => ""}
+            %td.narrow
+              %input{"type"        => "checkbox",
+                     "name"        => "cb_deletebacking",
+                     "ng-model"    => "disk.cb_deletebacking",
+                     "ng-if"       => "disk.add_remove!='add'",
+                     "ng-disabled" => "disk.add_remove=='remove'"}
+            %td
+              %button.btn.btn-danger.btn-sm{:type      => "button",
```

Comment:
@epwinchell - I used a red button here because  this is what the trello card indicated for the remove button. I will change it to default.